erpnext/stock/doctype/delivery_note/delivery_note.py

```
@@ -975,7 +976,7 @@ def select_item(d):
 
 
 @frappe.whitelist()
-def make_delivery_trip(source_name, target_doc=None, kwargs=None):
+def make_delivery_trip(source_name: str, target_doc: Document | None = None, kwargs: dict | None = None):
```

Ruff (0.15.1)

[warning] Unused function argument: `kwargs`

(ARG001)